config/samples/druid_v1alpha1_etcd.yaml

```diff
@@ -33,7 +32,7 @@ spec:
         clientPort: 2379
         serverPort: 2380
     backup:
-        version: 0.8.0-dev
+        version: 0.8.0-dev-dd26b65a2ab4efd77805584b33e8c559dd3c6d53
```

Comment:
Is this change required?

Reply:
Until we make a release of etcd-backup-restore, we need to use this image because it contains API changes to etcdbrctl.

Reply:
As Shreyas said, we need this till the changes are released.